example/cmd/device-simple/res/configuration.toml

```diff
@@ -37,6 +37,27 @@ Type = 'consul'
   Host = 'localhost'
   Port = 48081
 
+[MessageQueue]
+Enabled = false
+Protocol = 'tcp'
+Host = 'localhost'
+Port = 1883
+Type = 'mqtt'
+PublishTopicPrefix = 'edgex/events' # /<device-profile-name>/<device-name>/<source-name> will be added to this Publish Topic prefix
+[MessageQueue.Optional]
+# Default MQTT Specific options that need to be here to enable environment variable overrides of them
+# Client Identifiers
+ClientId = "device-simple"
+# Connection information
+Qos = "0" # Quality of Sevice values are 0 (At most once), 1 (At least once) or 2 (Exactly once)
+KeepAlive = "10" # Seconds (must be 2 or greater)
+Retained = "false"
+AutoReconnect = "true"
+ConnectTimeout = "5" # Seconds
+SkipCertVerify = "false" # Only used if Cert/Key file or Cert/Key PEMblock are specified
+ClientAuth = "none" # Valid values are: `none`, `usernamepassword` or `clientcert`
+Secretpath = "messagebus"  # Path in secret store used if ClientAuth not `none`
```

Comment:
This section still need to be indented.

Reply:
not sure how to indent this, is the following indention correct?
```
ClientId      = "device-simple"
Qos           = "0"
KeepAlive     = "10"
Retained      = "flase"
AutoReconnect = "true"
...
```